modules/configuring/proc-using-the-operator-to-run-rhdh-with-your-custom-configuration.adoc

```diff
@@ -9,9 +9,9 @@ To use the {product-short} Operator to run {product} with your custom configurat
 * Injects environment variables provisioned in your custom secrets.
 
 .Prerequisites
-* By using the {ocp-docs-link}/html-single/cli_tools/index#cli-about-cli_cli-developer-commands[{openshift-cli}], you have access, with developer permissions, to the {ocp-short} cluster aimed at containing your {product-short} instance.
-* {installing-on-ocp-book-link}[Your {ocp-short} administrator has installed the {product} Operator in {ocp-short}].
-* xref:provisioning-your-custom-configuration[You have provisioned your custom config maps and secrets in your `_<{my-product-namespace}>_` project].
+* By using the {platform-cli-link}, you have access, with developer permissions, to the {platform} cluster aimed at containing your {product-short} instance.
+* Your {ocp-short} administrator has installed the {product} Operator in {ocp-short}.
```

Comment:
```suggestion
* Your administrator has installed the {product} Operator in the cluster.
```

This is supposed to be common to all platforms, right? I guess it should be generic or use the right platform.. Otherwise, it is a bit weird to see this in the [AKS docs](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-1382/install-rhdh-aks/#using-the-operator-to-run-rhdh-with-your-custom-configuration) for instance:
<img width="807" height="211" alt="image" src="https://github.com/user-attachments/assets/03a9abe2-31d6-4738-b81b-f76ed98af415" />